packages/vite/package.json

```diff
@@ -1,18 +1,34 @@
 {
   "name": "vite",
   "version": "3.0.0-alpha.1",
+  "type": "module",
   "license": "MIT",
   "author": "Evan You",
   "description": "Native-ESM powered web dev build tool",
   "bin": {
     "vite": "bin/vite.js"
   },
-  "main": "dist/node/index.js",
-  "types": "dist/node/index.d.ts",
+  "main": "./dist/node/index.js",
+  "module": "./dist/node/index.js",
```

Comment:
I think we could remove these 2 fields since Vite 3 supports Node 14+, Node 12+ supports `exports` field and `exports` takes priority than `main` and `module` so Node.js will never reach these 2 fields.